frontend/src/app/output/page-content/open-api-end-points/open-api-end-points.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { OpenApiEndPointsComponent } from './open-api-end-points.component';
+
+describe('OpenApiEndPointsComponent', () => {
+  let component: OpenApiEndPointsComponent;
+  let fixture: ComponentFixture<OpenApiEndPointsComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ OpenApiEndPointsComponent ]
+    })
+    .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(OpenApiEndPointsComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+});
```

Comment:
Can you add some tests ? Not sure if it's possible to test Swagger display in nominal case, but you should at least be able to test error message display when it fails.